CMakeLists.txt

```diff
@@ -203,6 +204,15 @@ if(ENABLE_SERVICES)
   endif()
 endif()
 
+# gdal
+set(gdal_target "")
+if (ENABLE_GDAL)
+  find_package(GDAL REQUIRED)
+  add_compile_definitions(ENABLE_GDAL)
+  set(gdal_target GDAL::GDAL)
+endif()
```

Comment:
apparently a FindGDAL module is only available since cmake 3.14. it's not at all a problem to bump the cmake version, 3.12 is ancient. another option would be to use pkg-config (see e.g. `zlib`, `spatialite` etc), which is my preferred option as then we're not relying on cmake versioned Find* modules which is really crap generally IMO, or on cmake-based package manager installations for some package-provided GdalConfig.cmake file (often not the case on debian based)

Reply:
Can you also bump the cmake minimum version to 3.14 @chrstnbwnkl ? Just to be consistent.